pephub/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.11.8"
+__version__ = "0.11.9"
```

Comment:
I would change version to 0.12 because we added history

Reply:
Done